src/adminCommerceActionsContent.js

```
@@ -1,23 +1,20 @@
 // STUB | ADMIN COMMERCE ACTIONS CONTENT
 
-// VARS
-// let code = ''
+// Initialize variables
 if (typeof code === "undefined") {
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)